test/e2e/gateway_api_test.go

```diff
@@ -51,8 +55,6 @@ var defaultRoutename = ""
 // feature gate is still in effect, preface the test names with "TestGatewayAPI"
 // so that they run via the openshift/release test configuration.
 func TestGatewayAPI(t *testing.T) {
-	t.Parallel()
```

Comment:
We discussed in slack that making this serial should not be necessary.

Reply:
https://github.com/openshift/cluster-ingress-operator/pull/1192#issuecomment-2705207947